Formula/gstreamer.rb

```diff
@@ -1,10 +1,37 @@
 class Gstreamer < Formula
   desc "Development framework for multimedia applications"
   homepage "https://gstreamer.freedesktop.org/"
-  url "https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.22.0.tar.xz"
-  sha256 "78d21b5469ac93edafc6d8ceb63bc82f6cbbee94d2f866cca6b9252157ee0a09"
-  license "LGPL-2.0-or-later"
-  head "https://gitlab.freedesktop.org/gstreamer/gstreamer.git", branch: "main"
+  license all_of: ["LGPL-2.0-or-later", "LGPL-2.1-or-later", "MIT"]
+  revision 1
+
+  stable do
+    url "https://gitlab.freedesktop.org/gstreamer/gstreamer/-/archive/1.22.1/gstreamer-1.22.1.tar.gz"
+    sha256 "e8fc50d9f68c83bea3c7f04d78de2a9d54511c69b3449a7df8e4e2edf80f1e00"
+
+    # When updating this resource, use the tag that matches the GStreamer version.
+    resource "rs" do
+      url "https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/archive/gstreamer-1.22.1/gst-plugins-rs-gstreamer-1.22.1.tar.gz"
+      sha256 "65a8d76fda5f8b4a4ce088c574d055a823e9955352e1c6415e644dca4a655bae"
+
+      # Fix build failure on macOS.
+      # TODO: Upstream this (if it works).
```

Comment:
Upstreamed at https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/issues/332. Restrictions on new accounts at freedesktop.org prevent me from sending an MR. Will update this PR with the link.